domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -82,6 +85,17 @@
                 return true;
             })));
 
+builder.Services.AddHttpClient<CvrClient>(c =>
+{
+    c.BaseAddress = new System.Uri("http://distribution.virk.dk");
+    //TODO Encrypt
+    c.SetBasicAuthentication("Energinet_Datahub_CVR_I_SKYEN", "b90251cd-8787-4b34-a516-f52b4451b893");
```

Comment:
Is this the actual username/password for Energinet to use for CVR?

Reply:
Yes - have not done secret yet